infra/build/functions/target_experiment.py

```diff
@@ -268,6 +287,10 @@ def main():
   parser.add_argument('--upload_build_log',
                       required=True,
                       help='GCS build log location.')
+  parser.add_argument('--upload_err_log',
```

Comment:
WHy don't we just do this by default?

Reply:
This takes the destination bucket dir, which is [different for each build](https://github.com/google/oss-fuzz-gen/pull/212/files#diff-0b3fc09b0cd7a72a95a428b53477f27c2f85c45c483f7bf6a4c8666233b8a887R393-R394).
It sets `required=False` to make a smooth transition to OSS-Fuzz-Gen, e.g., https://github.com/google/oss-fuzz/pull/11700#discussion_r1527658501.